pkg/sfu/receiver.go

```diff
@@ -204,16 +204,23 @@ func (w *WebRTCReceiver) AddUpTrack(track *webrtc.TrackRemote, buff *buffer.Buff
 	w.buffers[layer] = buff
 	w.bufferMu.Unlock()
 
-	if w.isSimulcast && w.useTrackers {
-		tracker := NewStreamTracker()
+	if w.Kind() == webrtc.RTPCodecTypeVideo && w.useTrackers {
```

Comment:
A couple of things here
- Use trackers for simple/simulcast (all video)
- Use very permissive values for base layer (i. e. we really want to say that it is alive). With screen sharing, there are very few frames per second at times. And with a 5 packet minimum, it declares that the layer is gone and takes a long time to recover. So, use layer specific parameters.

Reply:
ah yes! nice catch here!

Reply:
I think, we can also remove the `useTrackers` option. this was done to ensure ion-sfu code still worked. I think we'd always want to use the tracker.

Reply:
Cool. Will clean up in a subsequent PR